src/app/editor/form-generator/providers/field-control-group/mesh-control.class.ts

```diff
@@ -90,6 +90,9 @@ export class MeshControl<T extends NodeFieldType> {
      * all descendants.
      */
     reset(value: T): void {
+        if (this.meshField) {
```

Comment:
This is now doing 2 things - resetting and also updating the value. I think the "force" argument could be removed from the `reset()` method and instead we just call `checkValue()` followed by `reset()`.